python/cuml/dask/feature_extraction/text/tfidf_transformer.py

```diff
@@ -167,6 +166,8 @@ def fit(self, X):
 
         wait_and_raise_from_futures([models])
 
+        models = models.result()
```

Comment:
Is this line missing from these two specific estimators or should it be added in other models as well (e.g. Logistic regression)?